src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -34,7 +35,7 @@ export const VideoView: React.FC = () => {
   const { state, updateWatchedVideos } = usePersonalData()
   const timestampFromQuery = Number(useRouterQuery('time'))
   const { getAssetUrl } = useAsset()
-
+  const searchRouteMatch = useMatch({ path: absoluteRoutes.viewer.video(id) })
```

Comment:
I think we may need to adjust the name of the variable now, sorry :P

Reply:
ah fk didnt see